games-puzzle/lightsoff/lightsoff-46.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{10..11} )
+inherit gnome.org gnome2-utils meson python-any-r1 xdg vala
+
+DESCRIPTION="Turn off all the lights"
+HOMEPAGE="https://wiki.gnome.org/Apps/Lightsoff"
+
+LICENSE="GPL-2+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+IUSE=""
```

Comment:
empty `IUSE=""` could be dropped

Reply:
Changed in https://github.com/gentoo/gentoo/pull/36859/commits/5cd5bcc8951009595a8d7559b10993b85b2e8c5d